project/src/components/add-review-button/add-review-button.tsx

```diff
@@ -1,5 +1,11 @@
-function AddReviewButton(): JSX.Element {
-  return <a href="add-review.html" className="btn film-card__button">Add review</a>;
+import { generatePath, Link } from 'react-router-dom';
+
+type AddReviewButtonProps = {
+  id: number;
+}
+
+function AddReviewButton({id}: AddReviewButtonProps): JSX.Element {
+  return <Link to={generatePath('/films/:id/review', {id: id})} href="add-review.html" className="btn film-card__button">Add review</Link>;
```

Comment:
href можно удалить, когда атрибутов больше 3 я обычно располагаю в столбик
```suggestion
  return <Link to={generatePath('/films/:id/review', {id})} className="btn film-card__button">Add review</Link>;
```